packages/perseus/src/index.ts

```diff
@@ -47,7 +47,7 @@ export * as constants from "./components/constants";
 export * as globalStyles from "./styles/global-styles";
 // TODO(FEI-4504): Replace globalStyles with wonder-blocks where possible
 export * as globalConstants from "./styles/global-constants";
-export * as icons from "./icon-paths";
+export {iconChevronDown, iconTrash} from "./icon-paths";
```

Comment:
I _wonder_ if we could just get rid of all these `icon-paths` and use http://phosphoricons.com 🤔  That might not be a simple replacement, but it might be relatively easy.

Reply:
Hmm, what would the benefit to moving to these phosphoricons over our existing implementation. It seems like this would require adding another dependency onto our codebase.

Reply:
We already have phosphor icons as [a dependency](https://github.com/Khan/perseus/blob/main/packages/perseus/package.json#L76) and WB uses them as the official source for icon paths. 

https://khan.github.io/wonder-blocks/?path=/docs/packages-icon-phosphoricon--docs